### PR TITLE
python3Packages.ds_store: init at 1.3.0

### PR DESCRIPTION
**Pull request by prusnak**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 108894` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python37Packages.ds_store</li>
    <li>python38Packages.ds_store</li>
    <li>python39Packages.ds_store</li>
  </ul>
</details>

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -1890,6 +1890,8 @@ in {
 
   dropbox = callPackage ../development/python-modules/dropbox { };
 
+  ds_store = callPackage ../development/python-modules/ds_store { };
```
Comment (fabaff) [inline pkgs/top-level/python-packages.nix:1893]: Please normalize the attribute name. According to the [Python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names).

```suggestion
  ds-store = callPackage ../development/python-modules/ds-store { };
```

The `pname` can remain as-is.

**Review by fabaff — COMMENTED** [review]
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] package build on (see https://github.com/NixOS/nixpkgs/pull/108894#issuecomment-757433378)
- [-] executables tested on ARCHITECTURE
- [-] all depending packages build
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.maintainers` is set
- [x] build time only dependencies are declared in `nativeBuildInputs`
- [x] source is fetched using the appropriate function
- [x] phases are respected
- [-] patches that are remotely available are fetched with `fetchpatch`

Otherwise, looks good to me.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108894` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>python37Packages.ds_store</li>
<li>python38Packages.ds_store</li>
<li>python39Packages.ds_store</li>
</ul>
</details>

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (prusnak) [reply pkgs/top-level/python-packages.nix]: Addressed in 7a3427d

**Comment by prusnak** [comment]
@GrahamcOfBorg build python3Packages.ds-store

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108894` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>python37Packages.ds_store</li>
<li>python38Packages.ds_store</li>
<li>python39Packages.ds_store</li>
</ul>
</details>
